fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -369,10 +375,132 @@ def create_report2(
     df_report = pd.DataFrame(reports)
     df_report = format_df(df_report)
     # Save & return
-    df_report.to_csv(CONFIG['outpath_report2'], index=False)
+    df_report.to_csv(CONFIG['outpath_report_thread_length'], index=False)
     return df_report
 
 
+def create_report_users(df: pd.DataFrame) -> Tuple[pd.DataFrame, pd.DataFrame, pd.DataFrame]:
+    """Report: Users
+    # TODO: Bugfix: Major bug; respondent/author counts are not all correct. This is because (i) threads are being
```

Comment:
@rohaher FYI: Major bugfix I need to do:
```
    # TODO: Bugfix: Major bug; respondent/author counts are not all correct. This is because (i) threads are being 
       counted multiple times when multiple keywords are matched against them, and (ii) we are *only* counting messages
       within threads that have keyword matches; not every message in every thread that has a keyword match for any 
       message.
```

Reply:
Related: #32